test/runtests.jl

```diff
@@ -11,71 +11,144 @@ import JSON
 import ForwardDiff
 
 const tests = [
-    "loguniform",
-    "arcsine",
-    "dirac",
+    "univariate/continuous/loguniform",
+    "univariate/continuous/arcsine",
+    "univariate/discrete/dirac",
     "truncate",
-    "truncnormal",
-    "truncated_exponential",
-    "truncated_uniform",
-    "truncated_discrete_uniform",
+    "truncated/normal",
+    "truncated/exponential",
+    "truncated/uniform",
+    "truncated/discrete_uniform",
     "censored",
-    "normal",
-    "laplace",
-    "cauchy",
-    "uniform",
-    "lognormal",
-    "mvnormal",
-    "mvlognormal",
-    "types",
+    "univariate/continuous/normal",
+    "univariate/continuous/laplace",
+    "univariate/continuous/cauchy",
+    "univariate/continuous/uniform",
+    "univariate/continuous/lognormal",
+    "multivariate/mvnormal",
+    "multivariate/mvlognormal",
+    "types", # extra file compared to /src
     "utils",
     "samplers",
-    "categorical",
+    "univariate/discrete/categorical",
     "univariates",
-    "continuous",
-    "edgecases",
-    "fit",
-    "multinomial",
-    "binomial",
-    "betabinomial",
-    "poissonbinomial",
-    "dirichlet",
-    "dirichletmultinomial",
-    "logitnormal",
-    "mvtdist",
-    "kolmogorov",
+    "univariate/continuous", # extra file compared to /src
+    "edgecases", # extra file compared to /src
+    "fit", # extra file compared to /src
+    "multivariate/multinomial",
+    "univariate/discrete/binomial",
+    "univariate/discrete/betabinomial",
+    "univariate/discrete/poissonbinomial",
+    "multivariate/dirichlet",
+    "multivariate/dirichletmultinomial",
+    "univariate/continuous/logitnormal",
+    "multivariate/mvtdist",
+    "univariate/continuous/kolmogorov",
     "edgeworth",
-    "matrixreshaped",
+    "matrixreshaped", # extra file compared to /src
     "matrixvariates",
-    "lkjcholesky",
-    "vonmisesfisher",
+    "cholesky/lkjcholesky",
+    "multivariate/vonmisesfisher",
     "conversion",
     "convolution",
-    "mixture",
-    "gradlogpdf",
-    "noncentralt",
-    "locationscale",
-    "quantile_newton",
-    "semicircle",
+    "mixture", # extra file compared to /src
+    "gradlogpdf", # extra file compared to /src
+    "univariate/continuous/noncentralt",
+    "univariate/locationscale",
+    "quantile_newton", # extra file compared to /src
+    "univariate/continuous/semicircle",
     "qq",
-    "pgeneralizedgaussian",
-    "product",
-    "discretenonparametric",
-    "chernoff",
-    "univariate_bounds",
-    "negativebinomial",
-    "bernoulli",
-    "soliton",
-    "skewnormal",
-    "chi",
-    "gumbel",
+    "univariate/continuous/pgeneralizedgaussian",
+    "multivariate/product",
```

Comment:
This file was probably wrongly moved. src/multivariate/product.jl contains deprecated code, whereas src/product.jl contains its successor. The tests here seem to refer to both. Shall I keep the original location or move it, as I presume src/product.jl will move into src/multivariate/?

Reply:
I think src/product.jl will not be moved as it defines multivariate, matrixvariate, and more general arrayvariate product distributions.

Reply:
Ok, I'll move the test file back then.